.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: node_js
+node_js:
+  - 4
+install:
+  - npm install bower -g
```

Comment:
Oh, we need bower to be installed globally unfortunately, as `psc-publish` can't find it otherwise. The `package.json` lists the local purescript and pulp dependencies for us.